internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-cloudfront-observability.yml

```diff
@@ -344,7 +344,7 @@ Resources:
         DefaultCacheBehavior:
           AllowedMethods: ["GET", "HEAD", "OPTIONS", "PUT", "PATCH", "POST", "DELETE"]
           CachePolicyId: 4135ea2d-6df8-44a3-9df3-4b5a84be39ad # See https://go.aws/3bJid3k
-          TargetOriginId: !Sub 'copilot-${AppName}-${EnvironmentName}-origin'
+          TargetOriginId: !Sub 'copilot-${AppName}-${EnvironmentName}'
```

Comment:
Will this introduce unexpected update for users with existing ALB+CDN setting?

Reply:
yeah i think so do you think we should keep the origin?